x-pack/plugin/analytics/src/test/java/org/elasticsearch/xpack/analytics/normalize/NormalizeAggregatorTests.java

```diff
@@ -160,7 +160,7 @@ private void testCase(ValuesSourceAggregationBuilder<?> aggBuilder, Consumer<Int
             termFieldType.setHasDocValues(true);
 
             try (IndexReader indexReader = DirectoryReader.open(directory)) {
-                IndexSearcher indexSearcher = newSearcher(indexReader, true, true);
```

Comment:
@polyfractal just pointed me to `AggregatorTestCase#newIndexSearcher` which seems to do that job. It has even less randomization, but it looks to be what we're moving the tests to.

Reply:
what is the reasoning behind maybeWrapping? and why is it unsupported in some scenarios?

Reply:
and thank you for the heads up. I've updated to use the new method